elfloader-tool/src/arch-riscv/linker.lds

```diff
@@ -4,6 +4,13 @@
  * SPDX-License-Identifier: GPL-2.0-only
  */
 
+/*
+ * NOTE: For RISC-V 32-bit, having the .rodata section before the .data section causes the elfloader to
```

Comment:
for readability, please break the line after 80 chars. Especially in comments this should be followed, as centuries of typography have shown that the average human prefers even less 70 chars per line.